test/integration/gardenlet/seed/seed/seed_test.go

```diff
@@ -581,6 +581,9 @@ var _ = Describe("Seed controller tests", func() {
 							},
 							Spec: operatorv1alpha1.GardenSpec{
 								VirtualCluster: operatorv1alpha1.VirtualCluster{
+									DNS: operatorv1alpha1.DNS{
+										Domain: "virtual-garden.local.gardener.cloud",
```

Comment:
Do we want to add a validation for the domain names? I guess there is not too much to validate, but we could if we want.

Reply:
Will do